pkgs/tools/networking/networkmanager/default.nix

```diff
@@ -52,19 +52,22 @@
   systemd,
   udev,
   udevCheckHook,
+  libnvme,
   withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemd,
+  # NBFT (NVMe Boot Firmware Table) support
+  withNbft ? libnvme != null,
```

Comment:
How does this work? Does `callPackage` _ever_ pass a `null` for `libnvme`?

Reply:
```suggestion
  withNbft ? false
```

This is the common way to make this an opt-in feature.

Reply:
Yeah that's what I expected, hence why I was confused seeing a null check like that.